tapdb/asset_minting_test.go

```diff
@@ -168,6 +171,85 @@ func storeGroupGenesis(t *testing.T, ctx context.Context, initGen asset.Genesis,
 	}
 }
 
+// AddRandomManagedUTXO is a helper function that will create a random managed
```

Comment:
nit: Capitalization of Godoc function name doesn't match actual function name.

Reply:
Good catch. I had it first defined someplace else.